portfolio/src/main/webapp/style.css

```diff
@@ -1,11 +1,14 @@
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  padding-top: 50px;
+  max-width: 650px;
+  width: auto;
```

Comment:
Please alphabetize declarations. Here and below. 

https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

Reply:
Done.